dll/win32/actxprxy/usrmarshal.c

```diff
@@ -200,6 +200,32 @@ HRESULT __RPC_STUB IEnumShellItems_Next_Stub(
     return hr;
 }
 
+HRESULT CALLBACK IEnumExplorerCommand_Next_Proxy(
```

Comment:
`#ifdef __REACTOS__` or submit to Wine?

Reply:
They don't have the `IExplorerCommand`, so I doubt they'll take this.

Reply:
`__REACTOS__` is also an acceptable solution to me; it just needs to stand out for future Wine syncs.

Reply:
In that case ill add the `__REACTOS__` definition, since the staging website seems a bit stale.
Thanks for your input!